### ISSUE TITLE
Scraper fails on block 16616983 transaction 242

### ISSUE THREAD

**Issue by tjayrush** [TrueBlocks/trueblocks-core#2748]
Because of this issue in Erigon.

https://github.com/ledgerwatch/erigon/issues/6956

Fixed in TrueBlocks with a hack and a TODO marker.

**Comment by tjayrush** [comment]
```
 rpcCall failed at block 16618114, tx 64 hash 0x484894104febe494f4bd04ea0482fcdf6d8e9bfbbdc66c2817fe556ba7b45f4a err empty hex string
 rpcCall failed at block 16618181, tx 146 hash 0x249e49da0744ad323a8f8edd10ab8e9851021caa6dbde8878cd881523f9ab26b err empty hex string
 rpcCall failed at block 16618196, tx 18 hash 0xd0700d48835234a91bb2e4b545acc46911b1696110350f67f7d376af72a7d4c3 err empty hex string
INFO[25-02|12:19:19.808] Pinning timestamp file /mnt/md0/trueblocks/v0.40.0/cache/mainnet//tmp/ts.ipfs_hash.fil to QmSEMFABg3aS1hDaA27pdKfnrmozZvSfU4X2Pq7dRHDbmT
INFO[25-02|12:19:19.836] Updating manifest with 3445 chunks
INFO[25-02|12:19:19.846] Wrote 337369 address and 2000573 appearance records to $INDEX/016612475-016618210.bin
INFO[25-02|12:19:19.846] Pinned chunk $INDEX/016612475-016618210.bin (QmQt3SaSRvp9oen1AY7CiVjbVg1RaysSmCod9kCGKN5Tyt,QmYehY5zzmDhDukwVH8tqybaundnTfY7jGWiQQndy51uBM)
INFO[25-02|12:19:20.099] Block=16619180 have 677489 appearances of 2000000 (33.9%). Need 1322511 more. Added 677489 records (338.74 apps/blk).
 rpcCall failed at block 16620098, tx 8 hash 0xcade66dabe53839af212edce080c9ae1e6669bb405f6cf167f874603751a6735 err empty hex string
 rpcCall failed at block 16620128, tx 12 hash 0x3884c225e16078b4cc98d725a7e615809afa1e45e514595d1b48624c2573501a err empty hex string
 rpcCall failed at block 16620182, tx 35 hash 0xaa3b54e05780e3e40c103ca6e4033d0e4a8429ee7183249f9d4e6a95d447becc err empty hex string
 rpcCall failed at block 16621080, tx 107 hash 0x32d576754610c7ddc6e2da0bf6a233f9584f50d66964ec06a884b6f814fa75bf err empty hex string
INFO[25-02|12:20:21.129] Pinning timestamp file /mnt/md0/trueblocks/v0.40.0/cache/mainnet//tmp/ts.ipfs_hash.fil to QmdqHDT9FB2ouNe2pJ4Y1hJDmo6J8JFdDtTCNH9qHL8Kdd
INFO[25-02|12:20:21.158] Updating manifest with 3446 chunks
INFO[25-02|12:20:21.172] Wrote 361308 address and 2000330 appearance records to $INDEX/016618211-016621018.bin
INFO[25-02|12:20:21.172] Pinned chunk $INDEX/016618211-016621018.bin (QmYD6y9xednAEPEwfjbiSSYtuMihyrekrvdXKsYpsPdBn3,QmY3mdnxCWNEKDBf3aEkmBxUXj6fyNBneVAdcAq5cPTXJB)
INFO[25-02|12:20:21.240] Block=16621180 have 127273 appearances of 2000000 (6.4%). Need 1872727 more. Added 127273 records (63.64 apps/blk).
 rpcCall failed at block 16621874, tx 130 hash 0x617562d74941db4f7418c6f3522f10ad199a7952adec33106839c8c33f497975 err empty hex string
INFO[25-02|12:21:12.231] Block=16623180 have 1469174 appearances of 2000000 (73.5%). Need 530826 more. Added 1341901 records (670.95 apps/blk).
 rpcCall failed at block 16623590, tx 179 hash 0xc8438b0c68312fe2d276e519b2354882466fdb505ac61ae6ec4e7165f912f384 err empty hex string
 rpcCall failed at block 16623602, tx 106 hash 0x7965e3999e190aff676cc148b373e1bd028185d3d297854d2f804a63cb2da3d8 err empty hex string
 rpcCall failed at block 16625022, tx 134 hash 0x26a50a895bc7f457a11b6187129976c60d9f298a4a404b00856c4cd2d9ba68ef err empty hex string
 rpcCall failed at block 16625022, tx 134 hash 0x26a50a895bc7f457a11b6187129976c60d9f298a4a404b00856c4cd2d9ba68ef err empty hex string
INFO[25-02|12:22:13.348] Pinning timestamp file /mnt/md0/trueblocks/v0.40.0/cache/mainnet//tmp/ts.ipfs_hash.fil to QmfL21SxVXfkEvDs2HmmNsENgLSwC1YYRX16vcxb18rYYY
INFO[25-02|12:22:13.370] Updating manifest with 3447 chunks
INFO[25-02|12:22:13.382] Wrote 353253 address and 2000204 appearance records to $INDEX/016621019-016623970.bin
INFO[25-02|12:22:13.382] Pinned chunk $INDEX/016621019-016623970.bin (Qma2WGdJ7M9ULSFifxdbKEuTFJGLMNUpizgvt9LvAGQeN8,QmTMAq51vjSLS8TDGGSLvBJbfFDJLhP9S1TbtPdNmJVsk4)
INFO[25-02|12:22:13.673] Block=16625180 have 751965 appearances of 2000000 (37.6%). Need 1248035 more. Added 751965 records (375.98 apps/blk).
 rpcCall failed at block 16626015, tx 131 hash 0x414417daa6f31cb6d3bcd1d7ecf16a40eac4713bd88166fb794b58fccd5987b1 err empty hex string
 rpcCall failed at block 16626181, tx 115 hash 0x57cff78a2acbc33defcbbc7d7fb33f0f02d8cc0a40008a4879e43671638db740 err empty hex string
 rpcCall failed at block 16627094, tx 167 hash 0xc2ffee98c24b496af0aa637ab0df45a1e9d36629bb1549370b84f7b4b6f4827b err empty hex string
INFO[25-02|12:23:12.864] Pinning timestamp file /mnt/md0/trueblocks/v0.40.0/cache/mainnet//tmp/ts.ipfs_hash.fil to QmXpFhKij1iLaVmFRL9Thx7i8FwEiHrCB7mFBecHqxp4qQ
INFO[25-02|12:23:12.898] Updating manifest with 3448 chunks
INFO[25-02|12:23:12.911] Wrote 400308 address and 2000329 appearance records to $INDEX/016623971-016627048.bin
INFO[25-02|12:23:12.911] Pinned chunk $INDEX/016623971-016627048.bin (QmcEXfspZLSLU18ebwLBndcAfJQDDMt2SEqSSh3NJiEJUV,QmTxC1pcxPanBoc47v5NF21NWYT5tLmX98AukmbdsEUqsh)
INFO[25-02|12:23:12.967] Block=16627180 have 90568 appearances of 2000000 (4.5%). Need 1909432 more. Added 90568 records (45.28 apps/blk).
 rpcCall failed at block 16627272, tx 125 hash 0x2646d6f946a791658b3ca302ea8fb4ea8c1063d9c643eac317d2b6fba5ecd235 err empty hex string
 rpcCall failed at block 16628102, tx 66 hash 0xb71b5d0e2c649a83ad70b408d5447dd603bed6b1adfd95b5cd65c4b5fdad1d87 err empty hex string
 rpcCall failed at block 16628171, tx 106 hash 0x5a2892161d9838089a404d8ad46db659ad3cf4ba77c633ddaa5b5a2591dd4134 err empty hex string
 ```

**Comment by tjayrush** [comment]
Seems to have possibly corrected itself by `016627049-016629735`.

**Comment by tjayrush** [comment]
This appears to be corrected at this point. Closing.
